contracts/Insurance.sol

```diff
@@ -199,7 +214,7 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
 
         uint256 ratio =
             PRBMathUD60x18.div(
-                getPoolTarget() - collateralAmount,
+                getPoolTarget() - collateralAmount - bufferAmount,
```

Comment:
unsure what this does.

Reply:
This ratio is `(insurance_fund_target - insurance_fund_holdings)/(leveraged_notional_value)` in the formula

Reply:
`collateralAmount` is the amount of quote in the public insurance pool and `bufferAmount` is the amount of quote in the buffer insurance pool

Reply:
This makes the calculation for funding rate equal to

`36523 * (10^11) * [(target - public collat amount - buffer collat amount) / leveraged notional value]`

Reply:
What's this for though? Insurance Funding Rate?

Reply:
Yep, that's right